### ISSUE TITLE
tf.keras.applications.mobilenet_v3.preprocess_input documentation not according source code 

### ISSUE THREAD

**Issue by SalvaPedraza** [tensorflow/tensorflow#45243]
In https://www.tensorflow.org/api_docs/python/tf/keras/applications/mobilenet_v3/preprocess_input
[Said ](https://www.tensorflow.org/api_docs/python/tf/keras/applications/mobilenet_v3/preprocess_input#returns) that:

Preprocessed numpy.array or a tf.Tensor with type float32.
The inputs pixel values are scaled between -1 and 1, sample-wise.

However in source code in GitHub you find this:

@keras_export('keras.applications.mobilenet_v3.preprocess_input')
def preprocess_input(x, data_format=None):  # pylint: disable=unused-argument
  return x

And if you check it in a notebook you see that as source code state it is doing nothing. Do not scale and do not change dtype to float32 of the input.

**Comment by qlzh727** [comment]
Thanks for reporting the issue.

To give more context, the preprocess_input() was a standard method for all models to normalize the input data. However, it is quite common that user forget to call this method before they feed the data to model, which result into poor result. When we realized this problem, we move the preprocessing logic from this method to the model itself for some new models (like mobilenet_v3). In order to keep the alignment between old and new model, we still keep this method for new model, but as an empty shell, so that user can easily switching between different version of models without changing much code.

I will update the docstring for this method to make it align with the implementation detail.

**Comment by mats-claassen** [comment]
Hi, the docs at https://www.tensorflow.org/api_docs/python/tf/keras/applications/mobilenet_v3/preprocess_input are still misleading (specifically, the `Returns` section)

For example, when trying to replicate the preprocessing based on the documentation to run it on mobile, this might lead to scaling the input values twice.

**Comment by qlzh727** [comment]
I think the doc change wasn't included in the 2.4 release, but will be reflect into 2.5 release (coming soon).

**Comment by MarkDaoust** [comment]
The nightly view shows what the current state is.

https://www.tensorflow.org/api_docs/python/tf/keras/applications/mobilenet_v3/preprocess_input?version=nightly

**Comment by yingshaoxo** [comment]
Something is wrong:

https://github.com/keras-team/keras/issues/14637#issuecomment-2246743316
